sdk/eventgrid/azure-eventgrid/setup.py

```diff
@@ -80,6 +81,7 @@
         'msrest>=0.5.0',
         'msrestazure>=0.4.32,<2.0.0',
         'azure-common~=1.1',
+        'azure-core~=1.6.0',
```

Comment:
That's not correct, this is forcing 1.6.x (1.7.0 is not allowed). Should be:
azure-core<2.0.0,>=1.7.0